presto-native-execution/presto_cpp/main/common/prometheus-metrics/PrometheusReporter.cpp

```diff
@@ -0,0 +1,172 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "PrometheusReporter.h"
+namespace facebook::presto::prometheus {
+void PrometheusReporter::registerMetricExportType(
+    const char* key,
+    facebook::velox::StatType statType) const {
+  if (registeredMetrics_.count(key)) {
+    // Already registered;
+    VLOG(1) << "Trying to register already registered metric " << key;
+    return;
+  }
+  // Prometheus format requires '.' to be replaced.
+  std::string keyStr = std::string(key);
+  std::replace(keyStr.begin(), keyStr.end(), '.', '_');
```

Comment:
I don't see the replacement of '.' being used in `addMetricValue` function do we need this there?